packages/flex-plugin-scripts/src/scripts/check-start.ts

```diff
@@ -182,6 +193,47 @@ export const _checkPluginCount = () => {
   }
 };
 
+/**
+ * First, touch ~/.twilio-cli/flex/plugins.json if it does not exist
+ * Next, checks if this plugin is in this config file. If not, add it.
+ * @private
+ */
+export const _checkPluginConfigurationExists = async() => {
+  // check if plugin.json exists
+  if (!checkFilesExist(paths.cli.pluginsJsonPath)) {
+      mkdirpSync(paths.cli.flex);
+      appendFileSync(paths.cli.pluginsJsonPath, JSON.stringify({plugins: []}, null, 2));
+  }
+
+  // templated read of package.json
+  const config = readJsonFile<CLIFlexConfiguration>(paths.cli.pluginsJsonPath);
+  const newPlugin = config.plugins.find((plugin) => plugin.name === paths.app.name);
+
+  // If plugin not found, add it
+  if (newPlugin === undefined) {
+    config.plugins.push({name: paths.app.name, dir: paths.app.dir, port: 0});
+    writeFileSync(paths.cli.pluginsJsonPath, JSON.stringify(config));
+    return;
+  }
+
+  // Plugin found with same directory
+  if (newPlugin.dir === paths.app.dir) {
+    return;
+  }
+
+  // Plugin found but with different directory
+  else {
```

Comment:
No `else` is required - you are already have `return` inside the `if` statement above